### ISSUE TITLE
How to make options modal appear below the icon

### ISSUE THREAD

**Issue by shazebq** [sampotts/plyr#1096]
The default behavior is that the options modal (e.g. speed options) appear above the player. However, this isn't really desirable in some cases such as when you're only using the audio player and there is no video. Is there any way to make the options modal appear below the player?

**Comment by friday** [comment]
Not really. It might be possible to override the current behavior with css, but this would require changing in a lot of places to get the arrow right too, and you'd likely need js to figure out how much to offset with (not sure about this). And ultimately it wouldn't really solve the problem imo.

As long as your page can be scrolled there's no telling which way is better until the menu is actually shown. Stuff like dynamic sorting and "infinite scroll" (tumblr scroll) makes it even harder. Changing the direction for all players may work for you if you know your player is always at the top on all devices, but even in that case it may overflow. It can happen for video too.

![screen shot 2018-05-31 at 22 04 25](https://user-images.githubusercontent.com/515120/42653713-86525546-8616-11e8-8bfc-062ef1c1bba8.png)

There's a whole library for handling this (https://popper.js.org/), but obviously adding that would increase the build size.
